src/preact.d.ts

```diff
@@ -37,10 +37,11 @@ declare namespace preact {
 		shouldComponentUpdate?(nextProps:PropsType,nextState:StateType,nextContext:any):boolean;
 		componentWillUpdate?(nextProps:PropsType,nextState:StateType,nextContext:any):void;
 		componentDidUpdate?(previousProps:PropsType,previousState:StateType,previousContext:any):void;
+		componentWillReplace?():void;
```

Comment:
I can't find any reference to `componentWillReplace` in our source code or `preact-compat`. Perhaps I'm missing something?